serving/docs/lmi/user_guides/embedding-user-guide.md

```diff
@@ -0,0 +1,981 @@
+  -0.21174066,
+  -0.18371966,
+  -0.11563054,
+  -0.15892655,
+  0.15574731,
+  -0.12771265,
+  0.027742216,
+  0.27574277,
+  -0.2625948,
+  -0.34240508,
+  0.8183645,
+  -1.2124608,
+  1.0034169,
+  -0.11373905,
+  -0.2432877,
+  0.24376862,
+  -0.47639894,
+  0.41671154,
+  -0.9653815,
+  0.4970341,
+  -0.11750264,
+  0.16793783,
+  -0.108437315,
+  -0.47287416,
+  -0.7300559,
+  0.1645548,
+  0.49839726,
+  -0.16526854,
+  0.17274109,
+  0.92254215,
+  0.8987754,
+  -0.4362364,
+  -1.0188268,
+  0.5145923,
+  0.20084324,
+  0.5625598,
+  -0.48971194,
+  -0.14288938,
+  -0.52283686,
+  -0.24525356,
+  0.6068987,
+  0.43570745,
+  -0.09429612,
+  0.41335583,
+  0.9717539,
+  0.2860962,
+  0.64698756,
+  -0.05734274,
+  0.54631644,
+  0.8332974,
+  0.54136103,
+  -0.4902781,
+  0.09807518,
+  -0.9091184,
+  0.4623836,
+  -0.63153464,
+  0.40226787,
+  -0.28529564,
+  -0.3228058,
+  -0.32955492,
+  -0.4590344,
+  -0.3084508,
+  0.7776922,
+  -0.055433553,
+  -0.34131756,
+  -0.656627,
+  -0.31522712,
+  -0.43529567,
+  -0.38323513,
+  0.8198578,
+  -1.2882359,
+  0.50641626,
+  0.80444396,
+  0.7380471,
+  0.60722286,
+  -0.19558027,
+  -0.45768735,
+  -1.1988037,
+  -1.4428236,
+  -0.64759225,
+  -0.6951825,
+  -0.066758275,
+  0.2753865,
+  0.44198763,
+  -0.5004722,
+  0.49417102,
+  0.40152887,
+  0.19156331,
+  -0.20195074,
+  -0.78548855,
+  -0.6010278,
+  0.43873742,
+  0.60128105,
+  0.009750693,
+  0.29560378,
+  0.15369748,
+  -0.11502031,
+  0.19575457,
+  -0.084062815,
+  -0.57200676,
+  0.52209103,
+  -0.8718636,
+  0.24354279,
+  -0.23001705,
+  -0.26813045,
+  0.033662252,
+  0.41768172,
+  -1.1656911,
+  0.16492313,
+  -0.23274249,
+  -0.09856258,
+  -0.14791532,
+  0.3483261,
+  -0.72370994,
+  -0.46186447,
+  1.002478,
+  0.06690536,
+  -0.6356916,
+  -0.8068857,
+  0.39828378,
+  -0.987602,
+  -0.48057964,
+  -0.91443294,
+  -1.1440288,
+  -0.09522212,
+  -0.36398962,
+  -0.14455658,
+  -0.09450873,
+  0.49147022,
+  0.37112617,
+  0.1446139,
+  -0.6551288,
+  -0.10683546,
+  1.0844564,
+  -0.26117444,
+  -0.61763823,
+  -1.0215288,
+  -0.47271717,
+  1.4268862,
+  0.51654065,
+  -0.01449356,
+  0.051638275,
+  0.20195235,
+  0.33145776,
+  -0.47064057,
+  0.03058975,
+  0.44798565,
+  0.043512702,
+  -0.5963866,
+  0.0999626,
+  -0.78542507,
+  -0.07645189,
+  -0.35353783,
+  -0.3817216,
+  -0.250955,
+  -0.71230185,
+  -0.7246541,
+  -0.09184249,
+  -0.15457578,
+  -0.47047973,
+  0.16415931,
+  0.4615652,
+  0.9969347,
+  1.1440169,
+  0.11543016,
+  -0.40084806,
+  0.8114938,
+  0.053596105,
+  0.072810955,
+  -0.48538062,
+  0.6282832,
+  -0.5842784,
+  -0.8101653,
+  0.7857535,
+  0.11797458,
+  -0.1375664,
+  -0.5850693,
+  0.60943407,
+  0.40796915,
+  -0.6434871,
+  0.3112606,
+  0.2279547,
+  -0.8009926,
+  -0.79564685,
+  0.37275973,
+  -0.48306096,
+  0.6792743,
+  0.7707857,
+  0.33682466,
+  0.6252336,
+  0.9911188,
+  0.18310043,
+  -0.5297898,
+  1.1664978,
+  0.79496926,
+  0.28931668,
+  1.2245488,
```

Comment:
this is too long, could we trim into ... in the middle? And also show the expected shape